core/ecschema-metadata/package.json

```diff
@@ -37,12 +37,15 @@
     "@bentley/build-tools": "2.15.0-dev.4",
     "@bentley/eslint-plugin": "2.15.0-dev.4",
     "@bentley/imodeljs-i18n": "2.15.0-dev.4",
+    "@bentley/units-schema": "^1.0.5",
+    "@types/almost-equal": "1.1.0",
     "@types/chai": "^4.1.4",
     "@types/chai-as-promised": "^7",
     "@types/mocha": "^5.2.5",
     "@types/node": "10.14.1",
     "@types/sinon": "^9.0.0",
     "@types/xmldom": "^0.1.29",
+    "almost-equal": "^1.1.0",
```

Comment:
this is listed as a dev dependency but it is used in the regular code.  It should be listed as a regular dependency.  The @types package should stay in dev dependencies because it's just used for typescript compile